src/sentry/static/sentry/app/views/settings/organization/permissionAlert.tsx

```diff
@@ -5,7 +5,10 @@ import {t} from 'app/locale';
 import Access from 'app/components/acl/access';
 import Alert from 'app/components/alert';
 
-const PermissionAlert = ({access, ...props}) => (
+type Props = React.ComponentPropsWithoutRef<typeof Alert> &
```

Comment:
I'm unfamiliar with the usage of `React.ComponentPropsWithoutRef`. Can you share more?

For components wrapped by `styled`, we can still use `innerRef` to access them. I'm unsure if styled-components or react-emotion have a type for such components.

Reply:
It just excludes the ref attribute, since including that in propTypes is often not what you want when you’re spreading props into something else